src/examples/mfem_common_plane.cpp

```diff
@@ -352,6 +352,8 @@ int main( int argc, char** argv )
     tribol::KINEMATIC_CONSTANT,
     tribol::NO_RATE_PENALTY
   );
+  // This set the penalty parameters on the contact meshes.
+  tribol::setMfemConstantPenalty(coupling_scheme_id, p_kine, p_kine);
```

Comment:
Can we keep names consistent with c-style API? So this would be setMfemKinematicConstantPenalty()? I like the clarity in distinguishing what is being penalized (kinematic gap or rate gap).